retrofit-mock/src/main/java/retrofit2/mock/BehaviorDelegate.java

```diff
@@ -55,7 +55,8 @@ public T returningResponse(@Nullable Object response) {
     return returning(Calls.response(response));
   }
 
-  @SuppressWarnings("unchecked") // Single-interface proxy creation guarded by parameter safety.
+  // @SuppressWarnings("unchecked") // Single-interface proxy creation guarded by parameter safety.
+  @SuppressWarnings("all")  //suprressing warnings for that are not resource leak checker. Ask Questions about this.
```

Comment:
If the warnings you're referring to here are actually not from the RLC, I wouldn't expect any difference between these two `@SuppressWarnings` annotations. Can you comment here with the error produced if you leave the existing `@SuppressWarnings("unchecked")` annotation?